clients/src/main/java/org/apache/kafka/clients/admin/KafkaAdminClient.java

```diff
@@ -4518,8 +4518,10 @@ private static byte[] getSaltedPassword(ScramMechanism publicScramMechanism, byt
     public DescribeFeaturesResult describeFeatures(final DescribeFeaturesOptions options) {
         final KafkaFutureImpl<FeatureMetadata> future = new KafkaFutureImpl<>();
         final long now = time.milliseconds();
+        final NodeProvider nodeProvider = options.nodeId().isPresent() ?
+            new ConstantNodeIdProvider(options.nodeId().getAsInt(), metadataManager.usingBootstrapControllers()) : new LeastLoadedBrokerOrActiveKController();
```

Comment:
I don't see why we need to use `metadataManager.usingBootstrapControllers()` instead of `true`. This request should be fully supported with the bootstrap controller

Reply:
Thanks for the review. Yes, this API supports bootstrap controller. Updated it.